modules/authentication/src/Authentication.ts

```diff
@@ -68,12 +69,32 @@ export default class Authentication extends ManagedModule<Config> {
   }
 
   async onConfig() {
-    if (!this.isRunning) {
+    const config = ConfigController.getInstance().config;
+    if (!config.active) {
+      this.updateHealth(HealthCheckStatus.NOT_SERVING);
+    } else {
+      let refreshedOnce = false;
       await this.registerSchemas();
+      if (config.local.verification.send_email) {
+        await this.grpcSdk.waitForExistence('email');
```

Comment:
wait for existence should be included in the email. I'm not saying that grpc-sdk shouldn't provide the waitForExistence function, but the monitor should use it, since it's to be expected.